CMakeLists.txt

```diff
@@ -48,6 +48,16 @@
 cmake_minimum_required(VERSION 3.24)
 project(executorch)
 
+# Enable ccache if available
+find_program(CCACHE_PROGRAM ccache)
+if(CCACHE_PROGRAM)
+  set(CMAKE_CXX_COMPILER_LAUNCHER "${CCACHE_PROGRAM}")
+  set(CMAKE_C_COMPILER_LAUNCHER "${CCACHE_PROGRAM}")
+  message(STATUS "ccache found and enabled for faster builds")
+else()
+  message(STATUS "ccache not found, builds will not be cached")
+endif()
+announce_configured_options(CCACHE_PROGRAM)
```

Comment:
oops you need to move this below line 66 cause we import this function later. I'd suggest to move this down to line 98 after we've configured some common requirements like python and buck